### PR TITLE
added templated for fhdds confirmation

### PR DESCRIPTION
**Pull request by YuanZHANG910**

### CODE REVIEW

File: app/uk/gov/hmrc/hmrcemailrenderer/templates/fhdds/fhddsBetaSubmissionConfirmation.scala.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
+@(params: Map[String, Any])
+@uk.gov.hmrc.hmrcemailrenderer.templates.helpers.html.template_main(params, "This is FHDDS Beta submission email."){
+<p style="margin: 0 0 30px; font-size: 19px;">
+    Dear Customer: <br><br>
+
```
Comment (drainhmrc) [inline app/uk/gov/hmrc/hmrcemailrenderer/templates/fhdds/fhddsBetaSubmissionConfirmation.scala.html]: Paragraphs can be encased in +< p style="margin: 0 0 30px; font-size: 19px;"> < /p>  tags then < br> won't look messy

File: app/uk/gov/hmrc/hmrcemailrenderer/templates/fhdds/fhddsBetaSubmissionConfirmation.scala.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    The facility to recover your username and/or password using information relating to your FHDDS application will not be available during the FHDDS Private Beta. This means that if you lose or forget your username and/or password, there may be significant delay in gaining access to the details of your application while the problem is sorted out.<br><br>
+    We therefore recommend that you use the same username and/or password that you already use to access other government services online in relation to your business. You should then be able to use information relating to one of these other services to recover your username and password if necessary.<br><br>
+    We expect the application process to be improved before being launched in April. We also expect more information about the due diligence requirements being agreed to by then.<br><br>
+
+    How to start testing:<br>
+    This link will take you straight to the test version of the service:<br>
+        @{params("verificationLink")}<br><br>
```
Comment (drainhmrc) [inline app/uk/gov/hmrc/hmrcemailrenderer/templates/fhdds/fhddsBetaSubmissionConfirmation.scala.html]: Confirm with whether If you are allowed to have a link in the email

File: app/uk/gov/hmrc/hmrcemailrenderer/templates/fhdds/fhddsBetaSubmissionConfirmation.scala.html [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    Please don't share this link with other people.<br>
+    We'll be monitoring how the service is used by invitees and may need to restrict access.<br><br>
+
+    How to give us your feedback:<br>
+    If you want to comment on any aspect of the service please email John Salisbury at john.salisbury@@digital.hmrc.gov.uk<br>
+    The more feedback you give us, the better we can make the next version of this service.<br><br>
+
+    How long testing lasts:<br>
+    You can submit an application any time during the next 5 days, but it would be really helpful to the team if you made a submission on the afternoon of 11th December 2017.<br>
```
Comment (drainhmrc) [inline app/uk/gov/hmrc/hmrcemailrenderer/templates/fhdds/fhddsBetaSubmissionConfirmation.scala.html]: This date needs to be updated or passed as a parameter

File: app/uk/gov/hmrc/hmrcemailrenderer/templates/fhdds/fhddsBetaSubmissionConfirmation.scala.html [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    After that we'll make and test a second version of the service based on the feedback we get.<br><br>
+
+    Why you got this email:<br>
+    This email is being sent to people who've been involved in our research so far, or who've volunteered to help us test this service.<br>
+
+    Thank you for your help and feedback,<br>
+    Karina Immanuel<br>
+    Digital Service Manager<br>
+    Fulfilment House Due Diligence Scheme<br>
+    If you’re unsure an email is from HMRC:<br>
```
Comment (drainhmrc) [inline app/uk/gov/hmrc/hmrcemailrenderer/templates/fhdds/fhddsBetaSubmissionConfirmation.scala.html]: the "if you're unsure ..." is done is the footer for the html

File: app/uk/gov/hmrc/hmrcemailrenderer/templates/fhdds/fhddsBetaSubmissionConfirmation.scala.txt [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+After that we'll make and test a second version of the service based on the feedback we get.
+
+Why you got this email:
+This email is being sent to people who've been involved in our research so far, or who've volunteered to help us test this service.
+
+Thank you for your help and feedback,
+Karina Immanuel
+Digital Service Manager
+Fulfilment House Due Diligence Scheme
+If you’re unsure an email is from HMRC:
```
Comment (drainhmrc) [inline app/uk/gov/hmrc/hmrcemailrenderer/templates/fhdds/fhddsBetaSubmissionConfirmation.scala.txt]: the "If you're unsure ..." can be done by 
`@{uk.gov.hmrc.hmrcemailrenderer.templates.helpers.txt.template_footer()}`

File: app/uk/gov/hmrc/hmrcemailrenderer/templates/fhdds/fhddsBetaSubmissionConfirmation.scala.txt [hunk 6]
```diff
@@ -0,0 +1,50 @@
+@(params: Map[String, Any])This is FHDDS Beta submission email.
```
Comment (drainhmrc) [inline app/uk/gov/hmrc/hmrcemailrenderer/templates/fhdds/fhddsBetaSubmissionConfirmation.scala.txt]: The changes in the text should match the changes in the html

File: app/uk/gov/hmrc/hmrcemailrenderer/templates/fhdds/fhddsBetaSubmissionConfirmation.scala.html [hunk 7]
```diff
@@ -0,0 +1,49 @@
+@(params: Map[String, Any])
+@uk.gov.hmrc.hmrcemailrenderer.templates.helpers.html.template_main(params, "This is FHDDS Beta submission email."){
```
[the remark this answers is not in the document]
Comment (drainhmrc) [reply app/uk/gov/hmrc/hmrcemailrenderer/templates/fhdds/fhddsBetaSubmissionConfirmation.scala.html]: guidelines number 4

**Comment by YuanZHANG910** [comment]
updated according to those commits and with the latest conffirmation messages. thanks for review

**Pull request merged**